src/Compilers/Core/Portable/AssemblyUtilities.cs

```diff
@@ -85,8 +82,7 @@ public static ImmutableArray<string> FindAssemblySet(string filePath)
         /// <exception cref="BadImageFormatException">If the file is not an assembly or is somehow corrupted.</exception>
         public static Guid ReadMvid(string filePath)
         {
-            Debug.Assert(filePath != null);
-            Debug.Assert(PathUtilities.IsAbsolute(filePath));
+            RoslynDebug.Assert(PathUtilities.IsAbsolute(filePath));
```

Comment:
Why is this necessary? Now that we're targeting `netcoreapp3.1` there shouldn't be any reason to use `RoslynDebug` over `Debug` that I'm aware of.

Reply:
This PR was started before you did the multi-targeting ;-)
Do you want me to remove all uses of RoslynDebug, or should we do that separately?

---
In reply to: [378074644](https://github.com/dotnet/roslyn/pull/41063#discussion_r378074644) [](ancestors = 378074644)

Reply:
Separately. 

I was more trying to work out how this could impact our flow analysis. Eventually I realized it was because it's `PathUtilities` (a type we define) not `Path` a type defined in SDK. Hence there was actual nullable information involved in this change.